src/main/java/mate/academy/model/Movie.java

```diff
@@ -0,0 +1,53 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movies")
+public class Movie {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private String description;
+
+    public Movie() {
+    }
```

Comment:
If you don't have a constructor with parameters, there is no need to override no-parameters constructor
```suggestion
    
```

Reply:
I know, but I'd like to use this as a fuse when I need to add a constructor

Reply:
Don’t create a default constructor when it is not needed. - from checklist )